jobs/e2e-test-cycloud.groovy

```diff
@@ -166,15 +166,25 @@ def runAllTestClients() {
 def runE2ETests(workerId) {
```

Comment:
What directory are we in when this function is run?

Reply:
If I’m not mistaken, the script executes the runE2ETests function within the webapp/services/static directory. This is defined in the dir('webapp/services/static') block in the parent function (runAllTestClients).

Reply:
OK, you don't want to create the tmpdir in webapp/services/static/results -- you want it to be at the top level.  According to my web research, you can do:
```
resultsDir = "${WORKSPACE}/results"; // Directory to store results
```
but you'll have to test that.

Also, "results" is a very generic name.  How about something like "junit-output/"?

Reply:
Thanks, Craig! {WORKSPACE} works, yeah and I’ll rename `results`! cc @michaelmendoza42